qa/zcash/performance-measurements.sh

```diff
@@ -77,10 +77,15 @@ case "$1" in
                 zcash_rpc zcbenchmark createjoinsplit 10
                 ;;
             verifyjoinsplit)
-                zcash_rpc zcbenchmark verifyjoinsplit 1000 "$RAWJOINSPLIT"
+                zcash_rpc zcbenchmark verifyjoinsplit 1000 "\"$RAWJOINSPLIT\""
                 ;;
             solveequihash)
-                zcash_rpc zcbenchmark solveequihash 10
+                if [ "$#" -ge "3" ]
+                then
+                    zcash_rpc zcbenchmark solveequihash 10 "$3"
+                else
+                    zcash_rpc zcbenchmark solveequihash 10
+                fi
```

Comment:
See http://stackoverflow.com/questions/9057387/process-all-arguments-except-the-first-one for a simpler way to do this.